src/Symfony/Bundle/FrameworkBundle/Command/CachePoolClearCommand.php

```diff
@@ -89,16 +89,25 @@ protected function execute(InputInterface $input, OutputInterface $output): int
             $clearer->clear($kernel->getContainer()->getParameter('kernel.cache_dir'));
         }
 
+        $failures = [];
         foreach ($pools as $id => $pool) {
             $io->comment(sprintf('Clearing cache pool: <info>%s</info>', $id));
 
             if ($pool instanceof CacheItemPoolInterface) {
-                $pool->clear();
+                if (!$pool->clear()) {
+                    $failures[] = $id;
+                }
             } else {
-                $this->poolClearer->clearPool($id);
+                if (false === $this->poolClearer->clearPool($id)) {
+                    $failures[] = $id;
+                }
             }
         }
 
+        if ($failures) {
+            throw new \Exception(sprintf('Cache pool "%s" could not be cleared.', implode('", "', $failures)));
```

Comment:
I think a warning + return 1 would be more appropriate to me

Reply:
done